d3xp/MultiplayerGame.cpp

```diff
@@ -3257,7 +3257,7 @@ void idMultiplayerGame::ClientStartVote( int clientNum, const char *_voteString
 	}
 
 	voteString = _voteString;
-	AddChatLine( va( common->GetLanguageDict()->GetString( "#str_04279" ), gameLocal.userInfo[ clientNum ].GetString( "ui_name" ) ) );
+	AddChatLine( "%s", va( common->GetLanguageDict()->GetString( "#str_04279" ), gameLocal.userInfo[ clientNum ].GetString( "ui_name" ) ) );
```

Comment:
and yea, this is wonky too and the va() (and the added "%s") could/should be removed